deployment/terraform/variables.tf

```diff
@@ -1,48 +1,50 @@
-variable "stack_name" {
-  default = "GeoTrellis Site"
-  type = "string"
-  description = "Disambiguation prefix for the EMR/ECS stack"
+variable "environment" {
+  default = "Production"
 }
 
-variable "service_image" {
-  type = "string"
-  description = "Geotrellis service Docker image"
+variable "remote_state_bucket" {
+  type        = "string"
+  description = "Core infrastructure config bucket"
 }
 
-variable "static_image" {
-  type = "string"
-  description = "Static service Docker image"
+variable "aws_account_id" {
+  default     = "896538046175"
+  description = "Geotrellis Site account ID"
 }
 
-variable "ec2_key" {
-  default = "geotrellis-cluster"
-  type = "string"
-  description = "EC2 key for EMR and ECS machines"
+variable "aws_region" {
+  default = "us-east-1"
 }
 
-variable "subnet_id" {
-  default = "subnet-c5fefdb1"
-  type = "string"
-  description = "Subnet ID shared by EMR and ECS"
+variable "git_commit" {
```

Comment:
A little bit of a nitpick, but I think keeping this more generic with `image_version` or `version` is better. Although it is a `git_commit`, it could be a bit misleading because it is also a container image tag.